runtime/doc/news.txt

```diff
@@ -200,6 +200,11 @@ The following new APIs or features were added.
   https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection .
   Support for the previous format will be removed in a future release.
 
+• Changes to |vim.treesitter.get_node_text()|:
```

Comment:
should be under "Changed features" below, no?

Reply:
Same for the above item, really.

Reply:
I'd say breaking changes. Wrong section either way.

Reply:
(actually, it looks like people just dumped everything under "new" (i.e., newly _added_) features. will have to clean up before release...)

Reply:
That works, too :D